web/src/app/services/ServiceLabelList.js

```diff
@@ -98,7 +98,10 @@ export default class ServiceLabelList extends React.PureComponent {
             <CardContent>{this.renderQuery()}</CardContent>
           </Card>
         </Grid>
-        <CreateFAB onClick={() => this.setState({ create: true })} />
+        <CreateFAB
+          onClick={() => this.setState({ create: true })}
+          title='create new service label'
```

Comment:
This should probably be `add service label` since labels aren't "created" in like other objects